suggestions/authenticators.json

```diff
@@ -4,5 +4,26 @@
         "label": "Aegis",
         "source": "fg",
         "repo": "https://github.com/beemdevelopment/Aegis"
+    },
+    {
+        "id": "me.jmh.authenticatorpro",
+        "label": "Authenticator Pro",
+        "source": "g",
+        "reason": "Open source Authenticator app featuring modern design, and compatibility with WearOS.\nNot available on F-droid yet because it is a Xamarin app (https://github.com/jamie-mh/AuthenticatorPro/issues/2#issuecomment-835407250)",
+        "repo": "https://github.com/jamie-mh/AuthenticatorPro"
+    },
```

Comment:
Authenticator app has two builds per releases :
- One with Wear OS support, which has non free dependencies
- One fully free of proprietary bits

**So Authenticator Pro has all its chances of belonging to this list!**

Reply:
From a security standpoint, Aegis is better than Authenticator Pro because the former has been here for a long time, has a [whitepaper](https://github.com/beemdevelopment/Aegis/blob/master/docs/vault.md) and received many formal and informal reviews. The choice of framework (Xamarin) by Authenticator Pro is also a major drawback since it consumes ~6 times more space than Aegis.

Reply:
https://www.usenix.org/conference/usenixsecurity23/presentation/gilsenan